docs/helpers.rst

```diff
@@ -130,8 +130,9 @@ Example
 
     from myapp.views import my_view
 
-    def test_details(rf):
+    def test_details(rf, admin):
         request = rf.get('/customer/details')
+        request.user = admin
```

Comment:
```suggestion
        # Remember that when using RequestFactory, the request does not pass
        # through middleware. If your view expects fields such as request.user
        # to be set, you need to set them explicitly.
        # The following line sets request.user to an admin user.
        request.user = admin
```

Reply:
This is a great addition :+1: 

I can vouch for that, have just spent some time finding out about how to use `django.contrib.messages` with RequestFactory.

Reply:
Looks good. Thank you.